spec/Task_Parallelism_and_Synchronization.tex

```diff
@@ -209,8 +209,7 @@ \section{Synchronization Variables}
 }
 \end{chapel}
 \begin{chapelpost}
-var tree: unmanaged Tree = new unmanaged Tree(false, new unmanaged
-Tree(false, new unmanaged Tree(true, nil, nil, 1),
```

Comment:
Could you please double-check that the next line fits on a page in the resulting PDF? If not, would you split it in a good spot?

Reply:
it doesn't show up in the PDF at all. It's part of the post-example-code for the test.